docs/source/analysis.ipynb

```diff
@@ -0,0 +1,530 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "**Welcome to Keypoint MoSeq Analysis Visualization Notebook!**\n",
+    "\n",
+    "This notebook contains a collection of analyses we typically perform on behavioral recordings to better understand mouse behavior. Examples of such analyses are:\n",
+    "- adding experimental group labels to the session\n",
+    "- descriptive labeling of individual syllables\n",
+    "- comparing syllable usage across different experimental groups\n",
+    "- comparing syllable transition structure across different experimental groups\n",
+    "\n",
+    "The goal of this notebook is to provide you with a basic summary of syllable usage and sequencing statistics across experimental conditions and to help generate more specific questions related to observations you may identify here.\n",
+    "If you want to perform more specific analyses, you can export the MoSeq data (see below) to use with the tool (i.e., Excel, Python, R, Matlab) you're most comfortable with.\n",
+    "\n",
+    "In this notebook, the **Markdown** above each cell describes the purpose of the cell(s), cell output, and the instructions for running the cell(s) and/or interacting with the widget. The **inline code comments** in the code block provides contextual information about the function, code structure, and parameters.\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Project setup\n",
+    "\n",
+    "## Files and Directory Structure\n",
+    "To run this notebook, you need the following files and folders in your data directory:\n",
+    "- `progress.yaml` - this file stores all the required paths used throughout the notebooks. The first time you run this notebook, a `progress.yaml` file will be created for you. If you have already run this notebook before, you can load the `progress.yaml` file by running the cell below.\n",
```

Comment:
copying @calebweinreb comments over:
I think that "progress.yaml" should live inside the model directory, since a person might want to run this analysis for multiple models

It should live in the base directory because the config file or index file is model agnostic.